src/core/content/connector.ts

```diff
@@ -150,7 +150,51 @@ export default class BaseConnector {
 	 * Has to be specified if usesBlocklist is set to true in connectors.ts.
 	 * If connectors.ts does not have usesBlocklist set to true, this should be null.
 	 */
-	public getChannelId: (() => string | null | undefined) | null = null;
+	public getChannelID: (() => string | null | undefined) | null = null;
```

Comment:
I know its a minor detail but I still think we should keep the casing of `Id` as it aligns with other things in the repo and the stdlib. e.g. `trackId`  `videoId` variable naming, `getElementById` etc.

Reply:
The issue is its inconsistent with `Connector.getUniqueID`, which AFAIK is the only public facing function we have using id.
Maybe we should rename that to `Connector.getUniqueId` in a future PR?